src/ankita/exam/Exam1.java

```diff
@@ -0,0 +1,39 @@
+package ankita.exam;
+
+import java.util.Arrays;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import ankita.base.PredefinedActions;
+
+public class Exam1 {
+
+	static void checkUserName(WebDriver driver) {
+		driver.findElement(By.linkText("Demo Tables")).click();
+		int rowsCnt = driver.findElements(By.xpath("//table[@id='table1']/tbody/tr")).size();
+		String[] arr = new String[rowsCnt];
```

Comment:
comparing data using array is good option but if can handle with simple way then don't go for the array (interview perspective)